pollux/vc-jwt/src/main/scala/org/hyperledger/identus/pollux/vc/jwt/Proof.scala

```diff
@@ -51,6 +51,9 @@ object Proof {
 }
 
 object EcdsaSecp256k1Signature2019ProofGenerator {
+  private def stripLeadingZero(arr: Array[Byte]): Array[Byte] = {
```

Comment:
I wonder if it's a dirty hack :)
Do we know the root cause or origin of different representations of the coordinates?

Reply:
Looks like encoded-key-to-coordinate conversion, [apollo](https://github.com/input-output-hk/atala-prism-apollo/blob/v1.3.5/apollo/src/commonMain/kotlin/org/hyperledger/identus/apollo/utils/KMMECSecp256k1PublicKey.kt#L90) should support this? Maybe we can also add support in `shared-crypto` [wrapper](https://github.com/hyperledger/identus-cloud-agent/blob/main/shared/crypto/src/main/scala/org/hyperledger/identus/shared/crypto/Apollo.scala#L59) as well if it's not too much work :smile: .